src/Workspaces/Core/Portable/Extensions/AdditionalProperty.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis
```

Comment:
I think this should live in the FindSymbols/FindUsages/FindReferences namespace. The property is specific to FAR reference and definition.